recipes-browser/cog/cog_0.6.0.bb

```diff
@@ -0,0 +1,7 @@
+require cog.inc
+
+SRC_URI = "https://github.com/Igalia/cog/releases/download/v${PV}/cog-${PV}.tar.xz"
+
+SRC_URI[sha256sum] = "208f58e0533b269400875237a95e994c93da7234a29fd1c904e756b88963e35d"
+
+RDEPENDS_${PN} += "wpewebkit (>= 2.28)"
```

Comment:
You may want to use the [Cog 0.8.0 stable release](https://wpewebkit.org/release/cog-0.8.0.html) instead of the old stable 0.6.x series.

Reply:
https://github.com/Igalia/meta-webkit/pull/197/commits/477926c66148a83ddf9750adda494c269d092a7d